doc/api_changes.rst

```diff
@@ -5,6 +5,23 @@ API changes
 Here we provide information about functions or classes that have been removed,
 renamed or are deprecated (not recommended) during different release circles.
 
+DIPY 0.16 Changes
+-----------------
+
+**Stats**
+
+Welcome to the new module ``dipy.viz.stats``. This module will be use to integrate various analysis.
```

Comment:
_(...)will be use to integrate various analysis._ -> _(...) will be used to integrate various analyses._